pkgs/development/libraries/openldap/default.nix

```diff
@@ -76,6 +80,7 @@ stdenv.mkDerivation rec {
   postInstall = ''
     make $installFlags install -C contrib/slapd-modules/passwd/sha2
     make $installFlags install -C contrib/slapd-modules/passwd/pbkdf2
+    make $installFlags install-lib -C contrib/slapd-modules/passwd/argon2
     chmod +x "$out"/lib/*.{so,dylib}
```

Comment:
Question: Why do those lib need the x bit set? Maybe we should add a comment to clarify this.

Reply:
Please look at the commit message in the PR - it states the reason. It's a bit too long to also have as a comment iMHO

Reply:
> openldap: Enable argon2 hash support by default …
>
>argon2 is the recommended password hashing function, and the module is
>included with OpenLDAP contrib.
>
>This change enables argon2 hashes by default in our OpenLDAP package.
>
>The install command for argon2 needs to be install-lib, as it otherwise
>tries to install manpages to /usr, which fails.

I don't think it mentions anything about chmod +x.

Reply:
That line was already there before this PR?

Reply:
Yeah it was. I am still curious why it is required.

Reply:
I don't know. I didn't try if it still works without the executable bit, I don't see why it shouldn't.